File: 2. Arrays/12. Wave Array/sol1.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;
vector<int> wave(vector<int> &A)
{
    sort(A.begin(),A.end());
    for (int i=0;i<(A.size()-1);i=i+2)
    {
      swap (A[i],A[i+1]);
    }
    return A;
}
int main ()
{
    vector<int> v{ 1, 5, 8, 9, 6, 7, 3, 4, 2, 0 } ;
    vector<int> wave_v=wave(v);
    for (auto x:v)
      cout << x << " ";
    cout << "\n";
    return 0;
}

File: 2. Arrays/12. Wave Array/sol1.py
class Solution:
    # @param A : list of integers
    # @return a list of integers
    def wave(self,a):
        a.sort()
        for i in range(0, len(a)-1, 2):
            a[i],a[i+1]=a[i+1],a[i]
        return a
